### DISCUSSION TITLE
[Projects Beta] Prioritize Exact Match For Repository Search When Adding Issues To Project Board

### DISCUSSION THREAD

**Discussion by haacked** [community/community#8517] [Projects and Issues] [answered]
Hello! When adding issues to the new Beta Project board, the filtering should prioritize exact match when ordering the possible matches. A screenshot is worth a thousand words here:

![Image showing adding new issue to project board](https://user-images.githubusercontent.com/19977/145087719-972b966e-766a-4a01-83c8-0ab27392bf1f.png)

In the screenshot above, I've typed `#abbot`. If I hit `TAB`, it will select the `abbot-skills` repository. But as you can see, there's an exact match of `abbot` in the list. I would expect that to be the one selected.

Thanks!

**Comment by azenMatt** [comment, marked as answer]
Hi @haacked - we have just made some updates here to prioritise the exact-name match when doing these searches so this should now bump abbot to the top. Let me know if you are still having problems with this! 😄

**Reply by haacked** [reply]
Works great! Thanks!

**Comment by Sadahmed443** [comment]
হাই @haacked[](https://github.com/haacked) - এই অনুসন্ধানগুলি করার সময় সঠিক-নামের মিলকে অগ্রাধিকার দিতে আমরা এখানে কিছু আপডেট করেছি তাই এটি এখন অ্যাবটকে শীর্ষে নিয়ে আসা উচিত। আপনি এখনও এই সঙ্গে সমস্যা হচ্ছে যদি আমাকে জানান!হাসি

১টি উত্তর
@হ্যাকড
[হ্যাক করা](https://github.com/haacked)
[১৯ দিন আগে](https://github.com/github/feedback/discussions/8517?sort=top#discussioncomment-2055499)
Author
দারুণ কাজ করে! ধন্যবাদ!

>
